matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DefaultCryptoService.kt

```diff
@@ -1301,6 +1308,35 @@ internal class DefaultCryptoService @Inject constructor(
         cryptoStore.logDbUsageInfo()
     }
 
+    override fun ensureOutboundSession(roomId: String) {
+        cryptoCoroutineScope.launch(coroutineDispatchers.crypto) {
+            // Ensure to load all room members
+            loadRoomMembersTask
```

Comment:
The idea to put it in this scope is to avoid using a callback and the taskExecutor.
Now you can directly write:
`loadRoomMembersTask.execute(LoadRoomMembersTask.Params(roomId))`

Reply:
Sure! 🆒